### PR TITLE
Remove reference to deprecated method getActionWithIdentifier

### PR DESCRIPTION
**Pull request by udayalla**

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/ios/push_notifications/action_buttons.md [hunk 1]
```diff
@@ -57,24 +57,24 @@ See our sample code [here][33] for `UserNotification.framework` and [here][32] f
...
 {% endtab %}
 {% tab swift %}
 
 ```swift
-Appboy.sharedInstance()?.getActionWithIdentifier(identifier,
-                         forRemoteNotification: userInfo,
-                             completionHandler: completionHandler)
+Appboy.sharedInstance()?.userNotificationCenter(center,
+                         didReceive: response,
+                             withCompletionHandler: completionHandler)
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/ios/push_notifications/action_buttons.md]: fix formatting

File: _docs/_developer_guide/platform_integration_guides/ios/push_notifications/action_buttons.md [hunk 2]
```diff
@@ -57,24 +57,24 @@ See our sample code [here][33] for `UserNotification.framework` and [here][32] f
 
 ## Step 2: Enable Interactive Push Handling
 
-To enable Braze's push action button handling, including click analytics and URL routing, add the following code to your app's `application:handleActionWithIdentifier:forRemoteNotification:completionHandler:` delegate method:
+To enable Braze's push action button handling, including click analytics and URL routing, add the following code to your app's `(void)userNotificationCenter:didReceiveNotificationResponse:withCompletionHandler:` delegate method:
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/ios/push_notifications/action_buttons.md]: Typically, someone following our normal push integration should already have this method integrated right? Might be worth calling that out

Comment (EmilyNecciai) [reply _docs/_developer_guide/platform_integration_guides/ios/push_notifications/action_buttons.md]: @udayalla I agree with @Bucimis here. When you reference that you should already have said method, I also suggest linking to the step in docs that first mentions that method in the initial push integration steps.

Comment (udayalla) [reply _docs/_developer_guide/platform_integration_guides/ios/push_notifications/action_buttons.md]: done!

File: _docs/_developer_guide/platform_integration_guides/ios/push_notifications/action_buttons.md [hunk 3]
```diff
@@ -57,24 +57,24 @@ See our sample code [here][33] for `UserNotification.framework` and [here][32] f
 
 ## Step 2: Enable Interactive Push Handling
 
-To enable Braze's push action button handling, including click analytics and URL routing, add the following code to your app's `application:handleActionWithIdentifier:forRemoteNotification:completionHandler:` delegate method:
+Typically, someone following our [normal push integration][39] should already have this method integrated. To enable Braze's push action button handling, including click analytics and URL routing, add the following code to your app's `(void)userNotificationCenter:didReceiveNotificationResponse:withCompletionHandler:` delegate method:
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/ios/push_notifications/action_buttons.md]: I think we can be more specific, e.g. "If you are using the UNNotification Framework and have implemented [Braze delegates][39], you should already..."

File: _docs/_developer_guide/platform_integration_guides/ios/push_notifications/action_buttons.md (hunk 1 above)
Comment (udayalla) [reply _docs/_developer_guide/platform_integration_guides/ios/push_notifications/action_buttons.md]: done!

**Pull request merged**